lost-paws/lost-paws
# Early map rendering, markers, API onboarding for Maps API

## Overview

**Issue Type**


**Description**
Early iteration of initial map render, early implementation of map markers on user input, some commenting to aid with onboarding of APIs

**Ticket/Card Name**

**Trello link - (https://trello.com/b/q3ghHlt1/lost-animal-tracker-app)**

**Steps to Validate Feature*

1. Go to localhost:5000, should see map render at initial user location as long as user elects to share their location

**Expected behavior**
Map to render a set initial position according to user current location, allow the addition of markers to the map on click at mouse position


**Screenshots & Videos**

![mapsAPI](https://user-images.githubusercontent.com/110871501/230740578-2d0d0113-8305-4e2a-afb9-a9a797db120d.JPG)

**Additional context**
Current concerns moving forward are persisting markers with frontend backend connection, assignment of information to markers, and render of information to markers. Also possible issues with querying markers relationally.
